libcaf_core/caf/actor_system_config.hpp

```diff
@@ -252,6 +252,13 @@ class actor_system_config {
   size_t work_stealing_relaxed_steal_interval;
   size_t work_stealing_relaxed_sleep_duration_us;
 
+  // -- config parameters for the logger ---------------------------------------
+
+  std::string logger_filename;
+  std::string logger_verbosity;
```

Comment:
You need to convert the atoms back to `uint64_t`. I've pushed a bit syntactic sugar for this with d729bd2af8b506eae1e9ddefe32c3d87307b9ad6.

```cpp
switch (to_u64(...)) {
  case to_u64(atom("NONE")):
    ...
}
```